contracts/core/RebalancingSetToken.sol

```diff
@@ -291,9 +301,21 @@ contract RebalancingSetToken is
         // Deposit newly created nextSets in Vault
         core.deposit(
             nextSet,
-            issueAmount
+            issueAmount.sub(totalFees)
         );
 
+        nextSetInstance.transfer(
+            manager,
+            managerFee
+        );
+
+        if (protocolFee > 0) {
+            nextSetInstance.transfer(
+                protocolAddress,
+                protocolFee
+            );
+        }
+
```

Comment:
Don't we need to update the unit shares after we transfer all the fees out? There is less withdrawable for everyone now

Reply:
Oh I see you did that in calculateNextSetIssueQuantity